### PR TITLE
[onert/test] Add Reader and View test

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit adds Reader and View gtest.

Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

File: runtime/onert/core/src/exec/feature/nhwc/View.h [hunk 1]
```diff
@@ -17,7 +17,7 @@
 #ifndef __ONERT_EXEC_FEATURE_NHWC_VIEW_H__
 #define __ONERT_EXEC_FEATURE_NHWC_VIEW_H__
 
-#include "../Reader.h"
+#include "Reader.h"
```
Comment (hseok-oh) [inline runtime/onert/core/src/exec/feature/nhwc/View.h:20]: Fix bug

File: runtime/onert/test/CMakeLists.txt [hunk 2]
```diff
@@ -10,6 +10,10 @@ target_link_libraries(${TEST_ONERT} onert_core)
 target_link_libraries(${TEST_ONERT} gtest)
 target_link_libraries(${TEST_ONERT} gtest_main)
 target_link_libraries(${TEST_ONERT} ${LIB_PTHREAD} dl)
+
+# Requires linking nnfw_coverage: check header coverage
+target_link_libraries(${TEST_ONERT} nnfw_coverage)
```
Comment (glistening) [inline runtime/onert/test/CMakeLists.txt:15]: (out of curiosity) Do we need this line for all onert tests?

**Review by glistening — APPROVED** [review]
LGTM

**Comment by glistening** [comment]
@chunseoklee Could you please take a look? I approved, but you pointed out negative tests are required in https://github.com/Samsung/ONE/pull/8971#issuecomment-1102547069. Is it okay w/o negative test? I found there is no negative tests in `ndarray` and other several files that I took a look to get a reference for negative tests. What is the exact rule? I vaguely remember that at least 2~3 years ago, the rule was # of negative tc >= # of positive tc.

**Comment by glistening** [comment]
Perhaps related to #8935

**Comment by chunseoklee** [comment]
> What is the exact rule?

There is no firm rule. We just keep negative test # > positive tests in repo. I try to apply it on pr review, but sometimes forgot :) 
For me, making meaningful negative test is difficult. In such cases, it is ok to merge wo neg test

**Comment by glistening** [comment]
@chunseoklee I talked with other colleague. I got similar explanation and his experiences. I updated #8971 with negative test.

**Comment by hseok-oh** [comment]
By this PR

- Total TC: 439
- Negative TC: 233

File: runtime/onert/test/CMakeLists.txt (hunk 2 above)
Comment (hseok-oh) [reply runtime/onert/test/CMakeLists.txt:15]: Maybe we need...

**Review by chunseoklee — APPROVED** [review]

**Pull request merged**